### PR TITLE
Standalone ROSETTA modelling

### PR DESCRIPTION
**Pull request by linucks**
This enables the AMPLE ROSETTA modelling package to be run in standalone mode to make it simple to create ROSETTA models in parallel using an arbitrary ROSETTA flagsfile and binary

### CODE REVIEW

File: ample/modelling/__main__.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+if args.rosetta_flagsfile is None:
+    raise RuntimeError("Need to supply a ROSETTA flagsfile with the -rosetta_flagsfile argument")
+flagsfile = args.rosetta_flagsfile
+if args.rosetta_AbinitioRelax is None:
+    raise RuntimeError("Need to supply a ROSETTA binary with the -rosetta_AbinitioRelax argument")
+flagsfile = os.path.abspath(args.rosetta_flagsfile)
+rosetta_binary = args.rosetta_AbinitioRelax
+
+nproc = args.nproc
```
Comment (hlasimpk) [inline ample/modelling/__main__.py]: Could set nproc default of 1 in the core options and remove this

File: ample/modelling/__main__.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    submit_array = True
+submit_max_array = args.submit_max_array
+
+
+nmodels = args.nmodels
+if nmodels is None:
+    nmodels = 1000
+work_dir = args.work_dir
+if work_dir is None:
+    work_dir = os.path.abspath('rosetta_modelling')
```
Comment (hlasimpk) [inline ample/modelling/__main__.py]: Most of these if statements could be replaced by the default flag in argparse, unless of course there is a good reason not to set a default?

**Review by hlasimpk — COMMENTED** [review]
Looks good, I've just made note of a few things which could have had defaults set.

**Comment by linucks** [comment]
Thanks @hlasimpk Useful comments. I'm going to leave the default setting as is as it's different from the main ample and I don't want to confuse things. However, you're comments made me realise I should use multiple processors on a workstation so thanks!

**Pull request merged**